### PR TITLE
chore : Added PR and Issue template

### PR DESCRIPTION
**Pull request by Hard-Coder05**
This PR solves the issue: https://github.com/openebs/openebs/issues/3144

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Steps to reproduce the bug should be clear and easily reproducible to help people gain an understanding of the problem
+
+**The output of the following commands will help us better understand what's going on**:
+<!-- (Pasting long output into a [GitHub gist](https://gist.github.com) or other [Pastebin](https://pastebin.com/) is fine.) -->
+
+* `kubectl get cstorvolume <pv-name> -n <openebs_namespace> -o yaml`
+* `kubectl get cstorvolumeconfig <pv-name> -n <openebs_namespace> -o yaml`
+* `kubectl logs sts/openebs-cstor-csi-controller -n kube-system -c openebs-csi-plugin`
+* `kubectl logs ds/openebs-cstor-csi-node -n kube-system -c openebs-csi-plugin`
+* `kubectl get pods -n <openebs_namespace>`
```
Comment (shubham14bajpai) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]:
```suggestion
* `kubectl get pods -n <openebs_namespace> --show-labels`
* `kubectl logs <upgrade_job_pod> -n <openebs_namespace>`
```

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+name: Feature request
+about: Suggest an idea to improve cstor-csi driver
```
Comment (shubham14bajpai) [inline .github/ISSUE_TEMPLATE/feature-request.md:3]:
```suggestion
about: Suggest an idea to improve openebs upgrades
```

**Review by shubham14bajpai — CHANGES_REQUESTED** [review]
Gave a few comments

**Review by shubham14bajpai — APPROVED** [review]
LGTM

**Comment by Hard-Coder05** [comment]
@shubham14bajpai @prateekpandey14 As the changes are aproved Please merge the PR!

**Review by prateekpandey14 — APPROVED** [review]
lgtm

**Comment by prateekpandey14** [comment]
Thank you @Hard-Coder05  for this PR. You can sign up for Hacktoberfest swag from MayaData by entering details here: https://github.com/mayadata-io/swag/issues/new/choose

**Comment by Hard-Coder05** [comment]
@prateekpandey14 I have a doubt! The contribution detail form had no option for T-Shirt size btw!

**Comment by prateekpandey14** [comment]
ahh , added the size option

**Pull request merged**